10_code/60_WA_and_controls_pop_merge.py

```diff
@@ -0,0 +1,79 @@
+import pandas as pd
+import numpy as np
+
+
+##import data
+pop = pd.read_csv(
+    "https://raw.githubusercontent.com/MIDS-at-Duke/pds-2022-red-team/main/20_intermediate_files/pop_merged.csv"
+)
+WA_pres = pd.read_csv(
+    "https://raw.githubusercontent.com/MIDS-at-Duke/pds-2022-red-team/wa_and_comparisons/20_intermediate_files/Opioid_Prescriptions_WA_comparisons.csv"
+)
+
+
+WA_col = ["WA", "OR", "CA", "NV", "ID", "MT"]
+
+# select only the states in Florida_col in population
+WA_pop = pop[pop["State"].isin(WA_col)]
+
+
+##find all county names ending with 'County' or 'Parish' except for Carson City
+WA_pop[
+    (~WA_pop["County"].str.contains("County"))
+    & (~WA_pop["County"].str.contains("Parish"))
+]
+
+
+##define condition when fips is 32510
+condition = WA_pop["fips"] == 32510
+
+##add string to values in column equal to 'A'
+WA_pop.loc[condition, "County"] = WA_pop["County"].astype(str) + " County"
+
+
+##remove "County" and "Parish" from the county names in the FL population data
+WA_pop["County"] = WA_pop["County"].str[:-7]
+
+
+##drop the first column and rename the columns
+WA_pres.drop(columns=["Unnamed: 0"], inplace=True)
```

Comment:
We can specify the columns we want when reading the csv file by passing `usecols = []` so that we don't have to drop it later. But that's just a personal preference.